src/helpers/getDataWithoutNestedAnchors.ts

```diff
@@ -0,0 +1,39 @@
+import {parse} from 'node-html-parser';
+import type {TemplateExecutor} from 'lodash';
+import type {Header} from '../types';
+
+const extractAnchorTags = (data: Header) => {
+  // Extract any existing anchor tags from the header variable
+  const matches = data.header.match(/<a[^>]*>.*?<\/a>/g);
```

Comment:
Also not sure on the performance of this calcualtions + regex lookp over complex html

Reply:
hmm I can try to parse html first and use querySelectorAll to check for anchor tags what do you think about that ?

Reply:
Need to check on this, as this might also have a rendering performance issue on gem page

Reply:
this dosen't check on the whole content its only check for small piece of HTML inside header

Reply:
no, please no regex here.